README.md

```diff
@@ -1,2 +1,10 @@
 # Scenarios
 This Repository is for solving the scenarios faced by students
+
+####
+If I understood well, then we must fork the Bahmni git repo ? 
```

Comment:
Pankaj This video is not for DevOps Challenges.
This is for the kind of the activites which you are asked to do at work/interview